tests/non_matching/coupling_08.with_umfpack=true.output

```diff
@@ -0,0 +1,41 @@
+
+DEAL::dim0: 1, dim1: 1, spacedim: 1
+DEAL::FE0             : FE_Q<1>(1)
+DEAL::FE1             : FE_Q<1>(1)
+DEAL::Dofs 0          : 9
+DEAL::Dofs 1          : 5
+DEAL::Constrained dofs: 0
+DEAL::Epsilon: 1.00000
+DEAL::Error on constants: 1.72490
```

Comment:
Can you please explain why these values are correct? Don't you expect to have a zero error?
If not, wouldn't it be better to test for a case in which the error vanishes or has some other special behavior?

Reply:
The error is close to zero only if you integrate exactly (which you can't do), and if you refine enough (which makes the test very slow). We tested with a more refined case, checked everything worked, then switched to coarse case, and just accepted the error we make. These cannot be exact.